cli/src/generate/dsl.js

```diff
@@ -279,6 +282,14 @@ function grammar(baseGrammar, options) {
     throw new Error("Grammar's 'name' property must not start with a digit and cannot contain non-word characters.");
   }
 
+  if (inherits && typeof inherits !== "string") {
+    throw new Error("Grammar's 'inherits' property must be a string.");
+  }
+
+  if (inherits && !/^[a-zA-Z_]\w*$/.test(name)) {
+    throw new Error("Grammar's 'inherits' property must not start with a digit and cannot contain non-word characters.");
```

Comment:
```suggestion
    throw new Error("The inheriting grammar's 'name' property must not start with a digit and cannot contain non-word characters.");
```

Also allowing the name to start with digits might be okay

Reply:
It would not be a valid identifier (not sure if it matters here but better safe than sorry).

Reply:
yeah that's fine, if it is ever an issue we can adjust then